README.md

```diff
@@ -44,185 +42,113 @@ _✨ A modern Anki [custom scheduling](https://faqs.ankiweb.net/the-2021-schedul
 
-For more detail on the mechanism of the FSRS algorithm, please see my papers: [A Stochastic Shortest Path Algorithm for Optimizing Spaced Repetition Scheduling (free access)](https://www.maimemo.com/paper/) and [Optimizing Spaced Repetition Schedule by Capturing the Dynamics of Memory (submit request)](https://www.researchgate.net/publication/369045947_Optimizing_Spaced_Repetition_Schedule_by_Capturing_the_Dynamics_of_Memory).
+The add-on has many useful features, you can read about it here: https://github.com/open-spaced-repetition/fsrs4anki-helper
 
-[FSRS4Anki Helper](https://github.com/open-spaced-repetition/fsrs4anki-helper) is an Anki add-on that supports the FSRS4Anki Scheduler. It has six features:
-1. **Reschedule** cards based on their entire review histories.
-2. **Postpone** due cards whose retention is higher than your target.
-3. **Advance** undue cards whose retention is lower than your target.
-4. **Balance** the load during rescheduling.
-5. **No Anki** on Free Days (such as weekends).
-6. **Disperse** Siblings (cards with the same note) to avoid interference & reminder.
+For more detail on the mechanism of the FSRS algorithm, please see my papers: [A Stochastic Shortest Path Algorithm for Optimizing Spaced Repetition Scheduling (free access)](https://www.maimemo.com/paper/) and [Optimizing Spaced Repetition Schedule by Capturing the Dynamics of Memory (submit request)](https://www.researchgate.net/publication/369045947_Optimizing_Spaced_Repetition_Schedule_by_Capturing_the_Dynamics_of_Memory).
 
-# Tutorial
+# Installation
 
 中文版请见：[FSRS4Anki 使用指北](https://zhuanlan.zhihu.com/p/636564830)
 
-## 1 Quick Start
 
-### 1.1 Enable Anki's V3 Scheduler
+## 1. Enable the V3 scheduler in Anki
 
-Preferences > Review > Enable V3 Scheduler
+Go to Tools > Preferences > Review > Enable V3 Scheduler.
 
-![image](https://github.com/open-spaced-repetition/fsrs4anki/assets/32575846/8f91fba8-9b8b-405c-8aa9-42123ba5faeb)
+![1](https://github.com/Expertium/fsrs4anki/assets/83031600/ecf7d77e-1244-4a0a-8cfd-22a5b76e0fe1)
 
-### 1.2 Paste FSRS Scheduler Code
+## 2. Export your deck or collection
 
-In the deck options, find the Advanced Settings column, and paste the code in [fsrs4anki_scheduler.js](https://github.com/open-spaced-repetition/fsrs4anki/releases/latest) into the Custom Scheduling field:
+Make sure to select “Include scheduling information” and “Support older Anki versions”.
 
-![image](https://github.com/open-spaced-repetition/fsrs4anki/assets/32575846/5c292f91-8845-4f8c-ac42-55f9a0f2946e)
+![2](https://github.com/Expertium/fsrs4anki/assets/83031600/66eb57b6-81ca-4ba5-a0d8-bada9daf9a59)
 
-Idealy, you've now started using the FSRS4Anki Scheduler. If you're unsure, you can change this part of the code:
+Additionally, if you are worried about privacy and don’t trust Google Collab, you can do the following before running the optimizer: 
```

Comment:
I don't think this paragraph is so important that it needs to be explained in detail in the guide.

Reply:
@Expertium, @user1823, what do you think of?

Reply:
You can put that part - about privacy and blanking out the fields - somewhere below, maybe in "Advanced features" or somewhere else.

Reply:
I prefer moving this part to wiki.

Reply:
In my opinion, this part is not important. Users can trust Google Colab. Rather, mentioning this would make users think that Google Colab can't be trusted to keep their data private. So, many of them would try to delete the contents of their notes before uploading to Colab, which would unnecessarily add to the time taken for setting up FSRS.